src/background.js

```diff
@@ -1,10 +1,15 @@
 window.scrNoti = window.scrNoti || {};
+const seenMessages = {};
+
+// TODO:
+// onQuit
+// mark read?
```

Comment:
Should marking a message as "unread" also send a message?

Reply:
My opinion is "no", but this is your "extended" feature, so you decide!

Reply:
I do not need it myself, that's why I have asked.

Well, then I will not include it (for now).